index.html

```diff
@@ -65,26 +78,28 @@
                 <div id="icon" class="md:w-3/4 max-w-[12rem]"></div>
               </div>
               <div class="flex items-center gap-2">
-                <span id="temp-min"
-                  >min.<span class="font-medium text-lg ml-1" data-value>0</span
+                <span id="temp-min font-paragraph"
```

Comment:
When I run the code, I get an error in the console and some details don't show up and not even the weather image.

![image](https://user-images.githubusercontent.com/65303812/225718106-771502e9-18cf-4646-acb5-8d672172fea5.png)

Reply:
My bad i didnt see it